docker_images/peft/requirements.txt

```diff
@@ -0,0 +1,13 @@
+starlette==0.27.0
+api-inference-community==0.0.30
+huggingface_hub==0.14.1
```

Comment:
This one might need an update.

Reply:
Missing this

Reply:
this is latest release version: https://pypi.org/project/api-inference-community/
do you want me to change huggingface_hub one?

Reply:
`huggingface_hub` latest is 0.15.1 https://pypi.org/project/huggingface-hub/

Reply:
Ah sorry, as @Narsil pointed out, we need a new version of the `api-inference-community` library as we changed `validation.py`.